app/views/groups/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <div id ="group-index-title">
```

Comment:
The categories page is not designed keeping the view of the orgignal design provided. For example three buttons are layed out in a way that they are user friendly and do not not follow user friendly UI design principle. Kindly revisit the design page and try to copy the same colors, layout, and fonts.

![btn_categories](https://user-images.githubusercontent.com/35504149/204051372-ce3adf04-fb02-43dd-8f6b-55b18134a924.png)

![categories](https://user-images.githubusercontent.com/35504149/204051395-d80e4687-be25-4c3b-adfe-9a310796fa63.png)